### PR TITLE
feat(criteria): Generate Criteria view

### PR DESCRIPTION
**Pull request by Jonazao**

### CODE REVIEW

File: src/constants/criteria/criteria-taste.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+const expectation = {
+  title: 'Expectations',
+  description: 'Based on the essence of the dish what is delivered',
+  examples: {
+    excellent: 'The price was lower than average, the flavor outstanding, it is just amazing',
```
Comment (Symphatico) [inline src/constants/criteria/criteria-taste.js:5]: Love this description

File: src/constants/criteria/criteria-taste.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
+const expectation = {
+  title: 'Expectations',
+  description: 'Based on the essence of the dish what is delivered',
```
Comment (Symphatico) [inline src/constants/criteria/criteria-taste.js:3]: Is it fine to add the price element to the 'essence of the dish'?

**Review by Symphatico — APPROVED** [review]
As they say there in my ranch, muydeli

File: src/constants/criteria/criteria-taste.js (hunk 1 above)
Comment (Jonazao) [reply src/constants/criteria/criteria-taste.js:5]: Let's keep descriptions generic

File: src/constants/criteria/criteria-taste.js (hunk 2 above)
Comment (Jonazao) [reply src/constants/criteria/criteria-taste.js:3]: Let's keep descriptions generic

**Pull request merged**